mono/metadata/object-offsets.h

```diff
@@ -183,7 +183,6 @@ DECL_OFFSET(MonoContext, wasm_bp)
 DECL_OFFSET(MonoContext, wasm_sp)
 DECL_OFFSET(MonoContext, llvm_exc_reg)
 
-DECL_OFFSET(MonoLMF, method)
```

Comment:
why was this removed? this breaks at least ARM (it likely others): https://github.com/mono/mono/blob/99198da52bee6d5a648a3e92923693bf56cb4d18/mono/mini/tramp-arm.c#L245

Reply:
nevermind, I didn't see the `ifdef` above.